### PR TITLE
Init ci workflow

### PR DESCRIPTION
**Pull request by AlexJoz**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Cython>=0.22
```
Comment (ibayer) [inline requirements.txt:1]: @AlexJoz 
Is this file still needed? The requirements are already listed in `pyproject.toml`.

Comment (AlexJoz) [reply requirements.txt:1]: @ibayer used in previous gitlab/docker setup. Probably we can export reqs from pyproject file on build step, but not related. Some sudden spaces.. don't need this file in current mr
